NEWS.md

```diff
@@ -16,6 +16,9 @@ New language features
   in `[A; B]` has always described concatenating along the first dimension (vertically), now two
   semicolons `[A;; B]` do so in the second dimension (horizontally), three semicolons `;;;` in the
   third, and so on. ([#33697])
+* A backslash (`\`) before a newline inside a string literal now escapes the newline while also
```

Comment:
I think it's unclear what "escapes" means here; sometimes escaping a character means *including* it, e.g. `"\""`.

There are also manual sections on string literals that should mention this.